### PR TITLE
Bug 1547066 - Trailhead modal styles

### PR DESCRIPTION
**Pull request by rlr**

### CODE REVIEW

**Comment by Mardak** [comment]
Just noting here before I forget… aaron wanted .onboardingTitle (for the cards) to have margin-bottom 4px instead of the current 10px.

**Comment by rlr** [comment]
> Just noting here before I forget… aaron wanted .onboardingTitle (for the cards) to have margin-bottom 4px instead of the current 10px.

Fixed in latest update.

File: content-src/asrouter/templates/Trailhead/_Trailhead.scss [hunk 1]
```diff
@@ -1,9 +1,19 @@
+
+
 .trailhead {
-  $benefit-icon-size: 64px;
-  $benefit-icon-spacing: $benefit-icon-size + 20px;
+  $benefit-icon-size: 62px;
+  $benefit-icon-spacing: $benefit-icon-size + 12px;
 
   background: url('#{$image-path}trailhead/accounts-form-bg.jpg') 100% 100%;
+  background-size: cover;
```
Comment (Mardak) [inline content-src/asrouter/templates/Trailhead/_Trailhead.scss]: The `100% 100%` is actually the positioning (not the size), and it should be `bottom`. Then you can put the `cover` size on the same line: `background: url(…) bottom / cover;`

File: content-src/asrouter/templates/Trailhead/_Trailhead.scss [hunk 2]
```diff
@@ -1,9 +1,19 @@
+
+
```
Comment (Mardak) [inline content-src/asrouter/templates/Trailhead/_Trailhead.scss]: -spaces?

File: content-src/asrouter/templates/Trailhead/_Trailhead.scss [hunk 3]
```diff
@@ -65,21 +80,53 @@
     }
 
     h3 {
-      color: $purple-40;
```
Comment (Mardak) [inline content-src/asrouter/templates/Trailhead/_Trailhead.scss]: can delete the purple definition

File: content-src/styles/_variables.scss [hunk 4]
```diff
@@ -75,6 +78,8 @@ $about-welcome-extra-links: #676F7E;
 $firefox-wordmark-default-color: #363959;
 $firefox-wordmark-darktheme-color: $white;
 $trailhead-purple: #2B2156;
+$trailhead-blue-60: #0250BB;
+$trailhead-start-hover-color: #B833E1;
```
Comment (k88hudson) [inline content-src/styles/_variables.scss]: This looks a bit jarring to me and inconsistent with the blue continue button, @aaronrbenson what do you think?

**Comment by k88hudson** [comment]
LOOKS AWESOME

**Pull request closed without merging**